samples/Samples.Mvc5/Controllers/HomeController.cs

```diff
@@ -131,6 +133,35 @@ public ActionResult About()
         /// <returns>The <see cref="ActionResult"/>.</returns>
         public ActionResult ResultsAuthorization() => View();
 
+        public async Task<ActionResult> CallAsync()
```

Comment:
this action simulates a typical async action that invokes an async service method (FooAsync)